types/tslint.json

```diff
@@ -0,0 +1,23 @@
+{
+    "extends": "@definitelytyped/dtslint/dtslint.json", 
+    "rules": {
```

Comment:
In case you haven't stumbled onto it yet, a lot of these have direct mappings to ESLint rules documented at https://typescript-eslint.io/rules/.

Reply:
Oh nice! Thank you! I had not found that! I can get rid of this then!

Reply:
Hmm so I think a lot of the tslint rules we have here are custom for TS declarations and don't have a eslint equivalent. In that case, we'd still need to run `dtslint` and extend `@definitelytyped/dtslint/dtslint.json` (which defines some of the rules we're disabling here). So I think any additions to eslint would be un-necessary